guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -100,22 +144,22 @@ endif::[]
 
 [leapp-foreman-plugins]
 name=Foreman plugins {ProjectVersion}
-baseurl=https://yum.theforeman.org/plugins/{ProjectVersion}/el8/$basearch
+baseurl=https://_yum.theforeman.org_/plugins/{ProjectVersion}/el8/$basearch
 enabled=1
 gpgcheck=0
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
 module_hotfixes=1
 
 [leapp-foreman-client]
 name=Foreman client {ProjectVersion}
-baseurl=https://yum.theforeman.org/client/{ProjectVersion}/el8/$basearch
+baseurl=https://_yum.theforeman.org_/client/{ProjectVersion}/el8/$basearch
 enabled=1
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman-client
 
 [leapp-puppet7]
 name=Puppet 7 Repository el 8 - $basearch
-baseurl=http://yum.puppetlabs.com/puppet7/el/8/$basearch
+baseurl=http://_yum.puppetlabs.com_/puppet7/el/8/$basearch
```

Comment:
are users meant to replace the URL to yum.theforeman.org? I don't think so, so I also think you should not make it italic. Is there any reason for this change? (also applies to all other URLs to yum.theforeman.org.

Reply:
I only now realized for example.com URLs it makes sense for italics. I took it too literally.